catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsFilterDelegateTest.java

```diff
@@ -2115,42 +2062,12 @@ private WfsFilterDelegate setupFilterDelegate(
   private void whenTextualStringType() {
     mockProps.add(MOCK_PROPERTY);
     when(featureMetacardType.getTextualProperties()).thenReturn(mockProps);
-    when(featureMetacardType.getAttributeDescriptor(MOCK_PROPERTY))
-        .thenReturn(
-            new FeatureAttributeDescriptor(
-                MOCK_PROPERTY, MOCK_PROPERTY, true, true, true, true, BasicTypes.STRING_TYPE));
+    when(featureMetacardType.isQueryable(MOCK_PROPERTY)).thenReturn(true);
   }
 
-  private void whenProperties(AttributeType<?> type) {
+  private void whenProperties() {
```

Comment:
✏️ Perhaps rename this method? It wasn't clear to me what this was doing by the name. Something like `setupMockProperties` maybe?

Reply:
Furthermore, do enough of the tests use this to warrant moving it to the `setUp` method? Would this cause us any issues?

Reply:
I just refactored these tests to make the function names clearer.